source.txt

```diff
@@ -204,7 +204,8 @@ Table of Contents
 
     All folders are treated as existing, and therefore GET requests to
     untouched folders SHOULD be responded to with a folder description
-    with no items (the items field set to '{}'). However, an empty
+    with no items (the items field set to '{}'). ETags of empty folders
+    MUST NOT change value for successive GET requests. However, an empty
     folder MUST NOT be listed as an item in its parent folder.
```

Comment:
>Why would that be such a big problem? Is this just about efficiency?

It doesn't matter to me. I chose a fixed value because it was an easier requirement to state, and relatively free of misinterpretations.

In an older commit I've given a more verbose (and explained) description of that requirement: https://github.com/untitaker/spec/commit/a01753d9d154d16e98b4220a0cd84c402ca192b8